src/lib/types/events.ts

```diff
@@ -9,6 +9,9 @@ export const FEATURE_REVIVED = 'feature-revived';
 export const FEATURE_IMPORT = 'feature-import';
 export const FEATURE_TAGGED = 'feature-tagged';
 export const FEATURE_TAG_IMPORT = 'feature-tag-import';
+export const FEATURE_STRATEGY_UPDATE = 'feature-strategy-update';
+export const FEATURE_STRATEGY_ADD = 'feature-strategy-add';
+export const FEATURE_STRATEGY_REMOVE = 'feature-strategy-remove';
```

Comment:
These new events needs to be available to all addons as well. Any idea on how to also apply them to existing addon config where the user may only subscribe to FEATURE_UPDATE?

Reply:
Should I make the service send both events for now, and we'll revisit this if/when we discuss events again?

Reply:
yes